lib/ruby_lsp/requests/support/rubocop_runner.rb

```diff
@@ -50,6 +50,9 @@ class ConfigurationError < StandardError; end
         sig { returns(T::Array[RuboCop::Cop::Offense]) }
         attr_reader :offenses
 
+        sig { returns(::RuboCop::Config) }
+        attr_reader :config_for_pwd
```

Comment:
Could we say `working_directory` rather than `pwd`?

Reply:
Sure, I'll do this after a new RuboCop release